homeassistant/components/matter/event.py

```diff
@@ -132,10 +138,11 @@ def _on_matter_node_event(
 DISCOVERY_SCHEMAS = [
     MatterDiscoverySchema(
         platform=Platform.EVENT,
-        entity_description=EventEntityDescription(
+        entity_description=MatterEventEntityDescription(
             key="GenericSwitch",
             device_class=EventDeviceClass.BUTTON,
             translation_key="button",
+            name_using_matter_labels=["label", "button"],
```

Comment:
is this on purpose all lowercase?

Reply:
The code that does label matching is now case insensitive - both the labels in that name_using_matter_labels list, as well as the FixedLabel values get converted to lower case. I did this because the standards don't really give much guidance on how a manufacturer should specify a label - some may be in lower case, upper, mixed, whatever. So I thought it made sense just to be case insensitive. 

Thus, the result would be the same if you wrote this as:
name_using_matter_labels=["Label", "Button"],